docker-compose.apps.yml

```
@@ -139,6 +139,28 @@ services:
+    volumes:
+      - file-dev-db-data:/var/lib/postgresql/data
+    profiles:
+      - infra
+    ports:
+      - "15532:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=file-dev-db
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)